javascript-loops-practice/add-suffix-to-all.js

```diff
@@ -0,0 +1,9 @@
+/* exported addSuffixToAll */
+function addSuffixToAll(words, suffix) {
+  const arr = [];
+  for (let i = 0; i < words.length; i++) {
+    words[i] = words[i] + suffix;
```

Comment:
be careful with this -- by reassigning `words[i]` you are mutating the original array. It would be better to just push `words[i] + suffix` to `arr`